src/backend/impl/src/repositories/types/user_profile.rs

```diff
@@ -59,17 +59,12 @@ mod tests {
     use super::*;
     use crate::fixtures;
     use rstest::*;
-    use rstest_reuse::*;
 
-    #[template]
```

Comment:
This is the only place `rstest_reuse` was used and I actually don't remember what I was trying to do here, but it's not doing anything useful anymore.